R/dsl-distributions.R

```diff
@@ -166,6 +195,8 @@ dsl_distributions <- local({
     distr_gamma_rate,
     distr_gamma_scale,
     distr_hypergeometric,
+    distr_negative_binomial_mu,
+    distr_negative_binomial_prob,
```

Comment:
```suggestion
    distr_negative_binomial_prob,
    distr_negative_binomial_mu,
```

if you want prob to be the default